Makefile.work

```diff
@@ -133,13 +133,17 @@ endif
 endif
 SLAVE_IMAGE = $(SLAVE_BASE_IMAGE)-$(USER_LC)
 
+ifneq ($(PLATFORM_ARCH),amd64)
+    override INCLUDE_FIPS := n
```

Comment:
>INCLUDE_FIPS [](http://example.com/codeflow?start=13&length=12)

How about `ENABLE_FIPS`?

Reply:
Sure, I will change to use ENABLE_FIPS.

Reply:
Is there a reason to use override here? If it's because FIPS modules aren't supported on anything besides amd64, then it's better to error out rather than silently disabling the FIPS option.

Reply:
Fixed, if not supported, print error message.